pinot-common/src/main/java/com/linkedin/pinot/common/utils/SegmentName.java

```diff
@@ -73,4 +74,30 @@ public int getPartitionId() {
   public String getPartitionRange() {
     throw new RuntimeException("No partitionRange in " + getSegmentName());
   }
+
+  public static boolean isHighLevelConsumerSegmentName(String segmentName) {
+    if (segmentName.endsWith(SEPARATOR) || segmentName.startsWith(SEPARATOR)) {
+      return false;
+    }
+
+    String[] parts =  segmentName.split(SEPARATOR);
+    if (parts.length != 3 && parts.length != 5) {
```

Comment:
May be add a comment on how the HLC/LLC segments look like, so it is more clear on why 3/4/5?